README.md

```diff
@@ -61,6 +61,14 @@ The following projects are official OpenJS Foundation projects. If you are inter
 * [Fastify](https://fastify.io)
 * [nvm](http://nvm.sh/)
 
+
+### OpenJS Collaboration spaces
+
+The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
+support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]
```

Comment:
It seems this link is blank.

Reply:
I think it's supposed to be: `[Collaboration Space Progression](./COLLABORATION_SPACE_PROGRESSION.md)`